### ISSUE TITLE
`since` not used with bittrex

### ISSUE THREAD

**Issue by shaund** [ccxt/ccxt#666]
`since` is never used. It looks like the bittrex api doesn't have a `since` field for this call, but this function should still parse it, since that's what the user would expect when passing a `since` timestamp. It looks the same for `limit`, but I haven't tested that one.

```
    def fetch_ohlcv(self, symbol, timeframe='1m', since=None, limit=None, params={}):
        self.load_markets()
        market = self.market(symbol)
        request = {
            'tickInterval': self.timeframes[timeframe],
            'marketName': market['id'],
        }
        response = self.v2GetMarketGetTicks(self.extend(request, params))
        return self.parse_ohlcvs(response['result'], market, timeframe, since, limit)
```
```
    def parse_ohlcvs(self, ohlcvs, market=None, timeframe='1m', since=None, limit=None):
        array = self.to_array(ohlcvs)
        return [self.parse_ohlcv(ohlcv, market, timeframe, since, limit) for ohlcv in array]
```
```
    def parse_ohlcv(self, ohlcv, market=None, timeframe='1d', since=None, limit=None):
        timestamp = self.parse8601(ohlcv['T'])
        return [
            timestamp,
            ohlcv['O'],
            ohlcv['H'],
            ohlcv['L'],
            ohlcv['C'],
            ohlcv['V'],
        ]
```

**Comment by kroitor** [comment]
> It looks like the bittrex api doesn't have a `since` field for this call

Yep, they don't offer any filtering

> but this function should still parse it

Which function should parse what? )

**Comment by shaund** [comment]
It currently returns all the candles available. It should filter the returned data to that starting at `since`. Otherwise, why have `since` if the returned data isn't going to be as expected?

**Comment by kroitor** [comment]
This is fixed in 1.10.244. `since` and `limit` are now handled correctly. Thx!
